### PR TITLE
WIP: e2e: run via go-run by default, GORUN_FIXTURE=0 for opt-out from Makefile

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
Blocked on https://github.com/golang/go/issues/40467.

**Comment by marun** [comment]
Is it necessary or desirable long-term that our e2e suite be responsible for starting servers? vs requiring test servers be started before the test run, which presumably could use go run and maybe not run into this issue?

File: test/e2e/framework/fixture.go [hunk 1]
```diff
@@ -137,6 +137,11 @@ func InProcessEnvSet() bool {
 	return inProcess
 }
 
+func LogToConsoleEnvSet() bool {
+	inProcess, _ := strconv.ParseBool(os.Getenv("LOG_TO_CONSOLE"))
```
Comment (marun) [inline test/e2e/framework/fixture.go:141]: nit: maybe rename the var to something either more or less specific?

Comment (marun) [inline test/e2e/framework/fixture.go:141]: nit: maybe rename the var to something either more or less specific?

**Pull request closed without merging**